src/explore-education-statistics-admin/src/pages/release/content/components/ReleasePreviewTableToolFinalStep.tsx

```diff
@@ -0,0 +1,87 @@
+import TableHeadersForm from '@common/modules/table-tool/components/TableHeadersForm';
+import TimePeriodDataTable from '@common/modules/table-tool/components/TimePeriodDataTable';
+import { FullTable } from '@common/modules/table-tool/types/fullTable';
+import { TableHeadersConfig } from '@common/modules/table-tool/types/tableHeaders';
+import { Publication } from '@common/services/publicationService';
+import DownloadTable from '@common/modules/table-tool/components/DownloadTable';
+import TableToolInfo from '@common/modules/table-tool/components/TableToolInfo';
+import Link from '@admin/components/Link';
+import { BasicPublicationDetails } from '@admin/services/publicationService';
+import React, { useEffect, useRef, useState } from 'react';
+
+interface ReleasePreviewTableToolFinalStepProps {
+  publication?: Publication | BasicPublicationDetails;
+  table: FullTable;
+  tableHeaders: TableHeadersConfig;
+}
+const ReleasePreviewTableToolFinalStep = ({
+  publication,
+  table,
+  tableHeaders,
+}: ReleasePreviewTableToolFinalStepProps) => {
+  const dataTableRef = useRef<HTMLElement>(null);
+  const [currentTableHeaders, setCurrentTableHeaders] = useState<
+    TableHeadersConfig
+  >();
+
+  useEffect(() => {
+    setCurrentTableHeaders(tableHeaders);
+  }, [tableHeaders]);
+
+  const getMethodologyLinks = () => {
+    // TODO SOW4 Build methodology links for the Publication
```

Comment:
This is the first problem I'm having trouble resolving. 

Property `publication` has changed to be a `Publication` or a `BasicPublicationDetails`. Both of those had a methodology field but on this branch `Publication` has a `methodologies` field rather than `methodology`.

We can't use properties `publication.methodology` or `publication.methodologies` here because the types don't have uniform properties.